src/transactions/FeeBumpTransactionFrame.cpp

```diff
@@ -194,8 +194,11 @@ FeeBumpTransactionFrame::commonValidPreSeqNum(AbstractLedgerTxn& ltx)
     auto v2 = bigMultiply(mInnerTx->getFeeBid(), getMinFee(lh));
     if (v1 < v2)
     {
-        getResult().feeCharged =
-            bigDivide(v2, mInnerTx->getMinFee(lh), Rounding::ROUND_UP);
+        if (!bigDivide(getResult().feeCharged, v2, mInnerTx->getMinFee(lh),
+                       Rounding::ROUND_UP))
+        {
+            getResult().feeCharged = UINT64_MAX;
```

Comment:
`feeCharged` is `int64_t`, not `uint64_t`.

Reply:
nice... I should have gotten a warning here